content/60.methods.md

```diff
@@ -2,179 +2,264 @@
+
+We received a large amount of interest from the broad community, beginning with coverage of the project by _Nature Toolbox_ [@doi:10.1038/d41586-020-00916-6] and an associated tweet about the project on April 1, 2020 [@url:https://twitter.com/j_perkel/status/1245454628235309057] (Figure @projectstats).
+Because the GitHub issues and comment systems are relatively similar to other common web activities, we found that authors were able to learn these tools fairly quickly.
+Similarly, the gitter chat also presented a low barrier to entry.
+The manuscript has continued to grow throughout the first year of the project, with both word count and the number of references continuing to trend upwards (Figure @fig:projectstats).
+Similarly, though only a fraction of potential contributors ended up contributing to the text included in the manuscripts (Figure @fig:projectstats), many of these contributors remained engaged over the course of a full year (Figure with dots).
+Additionally, new contributors have continued to join even as the project begins its second year.
+
+(Dot plot of contributions will go here) <!--To Do: Just adding a comment to add the caption and reference to figure here and also about in "Figure with dots". Also, I'm only counting 41 authors here, whereas 49 are mentioned earlier and later. Is that just because some folks contributed but did not use GitHub? If so, should note that.-->
+
+In order to make the project accessible to individuals from a range of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
+We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
+Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.
+Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.<!-- To Do: Number of paper issue and number of unique contributors?-->
+
+All seven manuscripts associated with this project will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
```

Comment:
```suggestion
All seven manuscripts associated with this project except for this one will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
```

Reply:
This is an eighth manuscript! (too many manuscripts)
```suggestion
The seven manuscripts produced by the consortium (beyond this one) will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews so that they can continue to be updated as more information becomes available.
```